introduction.md

```diff
@@ -3,3 +3,7 @@ Hello, I am eemike07.  I have entered the dojo of github.
 My favorite codeschool path is "mastering github" and also "git real".
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?